applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -1014,11 +1019,20 @@ static void disable_subscription(struct subscriber *sub, uint8_t report_id)
 			continue;
 		}
 
+		if (sub->enabled_reports_bm == 0) {
+			/* Subscriber disconnected. */
+			send_empty_hid_out_reports(per, sub);
+		}
+
 		drop_enqueued_reports(&per->enqueued_reports, irep_idx);
 	}
 
 	/* And also this subscriber. */
 	drop_enqueued_reports(&sub->enqueued_reports, irep_idx);
+
+	if (sub->enabled_reports_bm == 0) {
```

Comment:
We could consider improving naming to clearly distinguish data related to input and output reports (in separate PR).
It could be a bit confusing, because e.g. we have bitmasks related to both out and in reports.